pkg/server/handler.go

```diff
@@ -43,18 +43,40 @@ func NewMsgHandler(logger logging.Logger, amqp network.IAmqpReceiver, thingContr
 
 // Start starts to listen messages
 func (mc *MsgHandler) Start(started chan bool) {
+	err := mc.start(started, make(chan network.InMsg))
+	if err != nil {
+		mc.logger.Error(err)
```

Comment:
Should we continue, even if this fails? :thinking:

Reply:
I believe so... that, or handle it properly so that we can come back from that error (which I wouldn't know how to do that, hehe). Any thoughts @netoax @esdrasjnr @Igoraamc?